t/ui/15-comments.t

```diff
@@ -62,6 +61,32 @@ is($driver->find_element('h4.media-heading', 'css')->get_text(), "Demo wrote les
 
 is($driver->find_element('div.media-comment', 'css')->get_text(), "This is a cool test", "body");
 
+# URL auto-replace
+$driver->find_element('textarea',       'css')->send_keys('foo@bar foo#bar <a href="https://openqa.example.com/foo/bar">https://openqa.example.com/foo/bar</a>: http://localhost:9562 https://openqa.example.com/tests/181148 (reference http://localhost/foo/bar ) bsc#1234 boo#2345 poo#3456 t#4567 t#5678/modules/welcome/steps/1');
```

Comment:
How about putting that string into an inline document, so it can line break? That line of yours is looooong